api/MediaDevices.json

```diff
@@ -146,14 +146,16 @@
               "version_added": "72"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": false,
+              "notes": "See <a href='https://crbug.com/487935'>bug 487935</a>."
```

Comment:
This bug looks like it's finally moving forward. If so, I have the same objection I have to listing bugs generally. The time they're included after fixing tends to far exceed the time they're useful to readers.

Reply:
I remove the links when I'm the reviewer as in https://github.com/mdn/browser-compat-data/pull/17034 and also clean up stale links when I find them, as in https://github.com/mdn/browser-compat-data/pull/17061.

What lint or process would be required for you to be happy with bug links for cases like this one?

Reply:
You're assuming there should be something here. I'm saying that the cost benefit analysis of a link's value doesn't favor including bug links of any kind.

Reply:
Is there a guideline on this? No other reviewers have pushed back on links, and I've added a lot of them, pretty sure each of @queengooborg @Elchi3 @ddbeck @sideshowbarker have reviewed such changes from me.

The benefit is that developers can click through to the bug and star/subscribe. And the Chrome team uses stars as a prioritization signal. For BCD maintainers, it also makes it clear that it's not just false by oversight, and changes to it should be reviewed with some care.

Reply:
We don't have a guideline regarding bug links, but typically we have encouraged adding bug URLs.  Along with the benefits that @foolip has described above, adding bug links also has the benefit of telling web developers that some consideration (at the very least) has been given to implement a feature.  In other words, adding a bug link changes a "not supported" field to "not supported, but may possibly be implemented in the future".  Additionally, if the bug is fixed, a developer can see that with the link and open a BCD issue saying "hey, this feature has now been implemented so the tables should be updated" -- without it, there are no references that a developer can refer to.

In regards to the delay between a bug getting fixed and BCD updating the version number, this is something @foolip and I have been working hard on with the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project.